### ISSUE TITLE
Button should not use the `alt` attribute

### ISSUE THREAD

**Issue by leecalcote** [layer5io/layer5#7485]
### Description

Several interactive elements use attributes that do not belong to them:

Button alt attributes: The "Get Started" buttons include an alt attribute:

```HTML
<button class="..." id="get-started-2" aria-label="Signup for Layer5 Cloud" title="Get Started" alt="Signup for Layer5 Cloud">
```

### Expected Behavior
The alt attribute is exclusively for images (`<img>`, `<area>`, `<input type="image">`). It is completely invalid on a `<button>`. Since the button already correctly uses aria-label and title, the alt attribute should be removed.

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="24px" align="left" /><h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/GvB8SudhEOoq3JOvoLaoMs)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/buttons/community.webp" height="22px" align="left" />Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

**Comment by AbhiSahane0** [comment]
Hi @leecalcote, I would to work on this bug.

Thanks

**Comment by megha1807** [comment]
Hi @leecalcote ! I’d like to work on this issue. Could you please assign it to me?

**Comment by Daniel-1600** [comment]
Hi @leecalcote , I’d like to work on this issue. Could you please assign it to me?

**Comment by hortison** [comment]
Thanks much, folks! I _think_, but could very well be wrong that @AbhiSahane0 and @megha1807 have other issues in-hand (that you're working on) now, so I handed this to @Daniel-1600.

With this said, I could be mistaken. 

@saurabhraghuvanshii @AbhiSahane0 @megha1807 @Daniel-1600 Also, it's worth noting that it's OK to have multiple issues assigned and to be working through multiple at a time. I was just trying to spread the load around a little bit. There are many other issues to be raised in terms of site performance. In fact, I think @leecalcote is working on overhaul of the build time and build memory for the project now. Still, please go run this test - https://pagespeed.web.dev - and open any issues that you find.

**Comment by Daniel-1600** [comment]
Thanks @hortison! I’ll start working on this and open a PR soon.
